test/memory_tools/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+# Create the memory_tools library which is used by the tests.
+add_library(${PROJECT_NAME}_memory_tools SHARED memory_tools.cpp)
+if(APPLE)
+  # Create an OS X specific version of the memory tools that does interposing.
+  # See: http://toves.freeshell.org/interpose/
+  add_library(${PROJECT_NAME}_memory_tools_interpose SHARED memory_tools_osx_interpose.cpp)
+  target_link_libraries(${PROJECT_NAME}_memory_tools_interpose ${GMOCK_LIBRARIES})
+  list(APPEND extra_test_libraries ${PROJECT_NAME}_memory_tools_interpose)
+  list(APPEND extra_test_env
+    DYLD_INSERT_LIBRARIES=$<TARGET_FILE:${PROJECT_NAME}_memory_tools_interpose>)
+elseif(UNIX)
+  # On Linux like systems, add dl and use the normal library and DL_PRELOAD.
+  list(APPEND extra_test_libraries dl)
+  list(APPEND extra_test_env DL_PRELOAD=$<TARGET_FILE:${PROJECT_NAME}_memory_tools>)
+endif()
+list(APPEND extra_lib_dirs $<TARGET_FILE_DIR:${PROJECT_NAME}_memory_tools>)
+target_link_libraries(${PROJECT_NAME}_memory_tools ${extra_test_libraries})
+target_compile_definitions(${PROJECT_NAME}_memory_tools
+  PRIVATE "RCL_MEMORY_TOOLS_BUILDING_DLL")
```

Comment:
This `RCL` prefix hasn't been changed to `RCUTILS`.

The same for all the prefixes in the `memory_tools.hpp`.